src/scripts/main.js

```diff
@@ -1 +1,64 @@
 'use strict';
+
+const body = document.querySelector('body');
+const successHandler = message => {
+  body.insertAdjacentHTML('beforeend', `
+    <div data-qa="notification" class="success">
+      ${message}
+    </div>
+  `);
+};
+const errorHandler = message => {
+  body.insertAdjacentHTML('beforeend', `
+    <div data-qa="notification" class="warning">
+      ${message}
+    </div>
+  `);
+};
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', e => {
+    e.preventDefault();
+
+    if (e.button === 0) {
+      resolve('First promise was resolved');
+    }
+  });
+
+  setTimeout(reject, 3000, 'First promise was rejected');
+});
+const secondPromise = new Promise(resolve => {
+  document.addEventListener('mousedown', e => {
+    e.preventDefault();
+
+    if (e.button === 0 || e.button === 2) {
+      resolve('Second promise was resolved');
+    }
+  });
+});
+const leftClickPromise = new Promise(resolve => {
+  document.addEventListener('mousedown', e => {
+    e.preventDefault();
+
+    if (e.button === 0) {
+      resolve();
+    }
+  });
+});
+const rightClickPromise = new Promise(resolve => {
+  document.addEventListener('mousedown', e => {
+    e.preventDefault();
+
+    if (e.button === 2) {
+      resolve();
+    }
+  });
+});
```

Comment:
You can do it in one handler by sending 'e.button'  that clicked in the parameter, the same code